cypress/integration/ProcessorTest.ts

```diff
@@ -18,10 +15,17 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
 
       it("Sink processor", () => {
         const processorName: string = "Sink processor";
-        const action = ["Slack", "dev-action", "https://test.app.com/item"];
-        const filters = [
-          ["data.name", "String equals", "John"],
-          ["data.surname", "String equals", "White"],
+        const action = {
+          "Action type": "Slack",
+          "Channel": "dev-action",
+          "Webhook URL": "https://test.app.com/item",
+          "Icon Emoji": "Property not configured",
+          "Icon URL": "Property not configured",
+          "Username": "Property not configured",
+        };
+        const filters: { [key in "Key" | "Type" | "Value"]: string }[] = [
```

Comment:
I think that this definiton could not be interpated as an property of an object (`.Key` notation)

Reply:
not sure I understand, could you please try to clarify?

Reply:
@jomarko I have just highligths the definition of the `filters` variable which is mentioned in the previous notes. I am not sure but this definition says that you can use only `filtres["Key"][1]`, isn't it?